tests/model/component_import.cpp

```diff
@@ -201,14 +191,14 @@ TEST(ComponentImport, multipleImportAndParse)
 
     libcellml::PrinterPtr printer = libcellml::Printer::create();
     std::string a = printer->printModel(m);
-    EXPECT_TRUE((e1 == a) || (e2 == a));
+    EXPECT_EQ(e, a);
```

Comment:
just to check my understanding, because we now track imports in the models there is no longer any ambiguity in the order in which these get printed and hence no need to check both ordering options?

Reply:
Yep, that's it.